demos/GMW_demo/gmw_and.js

```diff
@@ -0,0 +1,174 @@
+
+/**
+ * AND on two shares, return secret share of ci.
+ * @function gmw_and
+ * @param {module:jiff-client~JIFFClient} jiff - the jiff instance
+ * @param {number} secret - the secret1 to share.
+ * @param {number} secret - the secret2 to share.
+ * @returns {object} a secret share of value ci
+ */
+
+function gmw_and(jiff,share1,share2) {
```

Comment:
This looks good. As we move it into the library, we may want to define the function on the SecretShare object so that it could be called as such `share1.gmw_and(share_2)` - although that doesn't change the logic so not high priority,